src/devices/SenseHat/SenseHatLedMatrixSysFs.cs

```diff
@@ -108,13 +108,9 @@ private void WriteColor(Color color)
             byte g = (byte)(color.G >> 2);
             byte b = (byte)(color.B >> 3);
             ushort col = (ushort)((r << 11) | (g << 5) | b);
-            Span<byte> encoded = stackalloc byte[2]
-            {
-                (byte)(col & 0xff),
-                (byte)(col >> 8)
-            };
 
-            _deviceFile.Write(encoded);
+            _deviceFile.WriteByte((byte)(col & 0xff));
+            _deviceFile.WriteByte((byte)(col >> 8));
```

Comment:
If possible, I would rather only calling Write once, as we used to.

Reply:
It's possible yes, but we would have to allocate a byte[]
I assumed WriteByte would write to the FileStreams buffer, since Flush is always called afterwards.

Reply:
we can allocate ahead of time though (i.e. in the constructor) and then only replace the bytes